config/config.md

```diff
@@ -305,123 +265,130 @@ Any keys not given will receive the default value.
 
 ```go
 {
```

Comment:
Hmm, so if this is trying to look like a json definition, I think we would want the tabs/spaces right?

Reply:
my md linter hated the tab, code block was also labelled ```go. fixed